lib/util/cmcd_manager.js

```diff
@@ -192,6 +207,24 @@ shaka.util.CmcdManager = class {
     }
   }
 
+  /**
+   * Apply CMCD data to a manifest request.
+   */
+  onPlaybackPlay() {
+    if (!this.playbackPlayTime_) {
+      this.playbackPlayTime_ = performance.now();
+    }
+  }
+
+  /**
+   * Apply CMCD data to a manifest request.
+   */
+  onPlaybackPlaying() {
+    if (!this.playbackPlayingTime_) {
+      this.playbackPlayingTime_ = performance.now();
```

Comment:
`performance.now()` is not available on all supported platforms. Please either switch to `Date.now()` or add some utility method which checks which one is supported.